pytorch_mppi/mppi.py

```diff
@@ -265,13 +265,16 @@ def _compute_total_cost_batch(self):
         self.perturbed_action = self._bound_action(self.perturbed_action)
         # bounded noise after bounding (some got cut off, so we don't penalize that in action cost)
         self.noise = self.perturbed_action - self.U
-        action_cost = self.lambda_ * self.noise @ self.noise_sigma_inv
+        action_cost = self.lambda_ * torch.abs(self.noise) @ self.noise_sigma_inv
```

Comment:
can you make this a parameter when creating the controller?  I'm not sure if the default value should be false, since we're changing behavior with the self.U instead of self.perturbed_action below